src/it/scala/org/zalando/spearheads/innkeeper/routes/PatchRoutesSpec.scala

```diff
@@ -0,0 +1,85 @@
+package org.zalando.spearheads.innkeeper.routes
+
+import akka.http.scaladsl.model.StatusCodes
+import org.scalatest.{BeforeAndAfter, FunSpec, Matchers}
+import org.zalando.spearheads.innkeeper.api.JsonProtocols._
+import org.zalando.spearheads.innkeeper.api._
+import org.zalando.spearheads.innkeeper.routes.AcceptanceSpecToken._
+import org.zalando.spearheads.innkeeper.routes.AcceptanceSpecsHelper._
+import spray.json._
+
+class PatchRoutesSpec extends FunSpec with BeforeAndAfter with Matchers {
+
+  private val newDescription = "new-description"
```

Comment:
I would put the private vals at the end of the class (as we already have them in other tests)
The motivation would be that the important stuff are the tests, and not the mock data.